app/commands/command/destroy/legislation.rb

```diff
@@ -15,7 +15,10 @@ def call
 
             r.litigations = []
             r.targets = []
-            r.save!
+
+            # Validation is false because it's possible that some precedent
+            # record (like geography) was removed earlier
+            r.save(validate: false)
```

Comment:
to be honest, do we need this save here at all? I know I've added it in the first PR from this series, but need to re-check it 😅

Reply:
I've just checked it, and it works without save :)

Reply:
nice!